A patch to ctags containing our changes to ctags
(changing anon tag names from anon<hash> to anon<number>).
diff --git a/ctags/main/parse.c b/ctags/main/parse.c
index 8fbb7148..c12f8662 100644
--- ctags/main/parse.c
+++ ctags/main/parse.c
@@ -4111,12 +4111,18 @@ extern void anonGenerate (vString *buffer, const char *prefix, int kind)
 	parser -> anonymousIdentiferId ++;
 
 	char szNum[32];
+#if 0
 	char buf [9];
 
 	vStringCopyS(buffer, prefix);
 
 	anonHashString (getInputFileName(), buf);
 	sprintf(szNum,"%s%02x%02x",buf,parser -> anonymousIdentiferId, kind);
+#else
+	/* we want to see numbers for anon functions in the tree view instead of the hash */
+	vStringCopyS(buffer, prefix);
+	sprintf(szNum,"%u", parser -> anonymousIdentiferId);
+#endif
 	vStringCatS(buffer,szNum);
 }
